### DISCUSSION TITLE
AE 里设置PAG填充模式不生效

### DISCUSSION THREAD

**Discussion by bonoyang1022** [Tencent/libpag#2190] [Q&A] [answered]
在AE 中设置占位图填充模式为剪裁，导出之后在 pagViewer 中没有生效，依旧显示为黑边。

Mac 版本：14.3.1
AE 版本：24.1.0
pagViewer 版本：4.3.6.102（stable）

**AE 设置：**
![AE 设置](https://github.com/Tencent/libpag/assets/41837951/3deea8fd-1b07-44c6-82f3-4ed6cec0a52f)

**预览效果：**
![预览效果](https://github.com/Tencent/libpag/assets/41837951/6303debd-f617-41d3-a6d3-9036b81f8138)

**AE 文件：**
[AE文件夹.zip](https://github.com/Tencent/libpag/files/14583287/AE.zip)

**Comment by kevingpqi123** [comment, marked as answer]
看了下原因，应该是导出插件的问题，导出后参数没有赋值，后续我们修复下导出插件
<img width="672" alt="image" src="https://github.com/Tencent/libpag/assets/16609221/396b8120-a57d-4263-9dbe-2d623b6ba2a3">

该参数可以通过终端 API 调整，可以先让开发同学通过 API 接口设置填充模式
